odometry_state_estimator_node:
  ros__parameters:
    lin_cutoff_freq: 1.0
    pub_invterval_sec: 0.05
    wheel_min_calibration_speed: 10.0
    wheel_calibration_sample_num: 20
    wheel_initial_corrections:
      front_left: 0.29624
      front_right: 0.29687
      rear_left: 0.31207
      rear_right: 0.31016
    collection_frequency: 10
    calibration_frequency: 100
    publish_odom: true